libdrgn/python/type.c

```diff
@@ -1808,67 +1863,73 @@ static DrgnType *compound_type(PyObject *tag_obj, PyObject *size_obj,
 DrgnType *struct_type(PyObject *self, PyObject *args, PyObject *kwds)
 {
 	static char *keywords[] = {
-		"tag", "size", "members", "qualifiers", "language", NULL,
+		"tag", "size", "members", "qualifiers", "template_parameters",
```

Comment:
Can we have `template_parameters` between `members` and `qualifiers`? Or does that break a bunch of tests?

Reply:
I think it only breaks one, so I'll fix it and change the order